lightning/src/ln/peer_handler.rs

```diff
@@ -781,7 +781,12 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 														continue;
 													}
 													msgs::DecodeError::InvalidValue => {
-														log_debug!(self.logger, "Got an invalid value while deserializing message");
+														reader.set_position(0);
+														let msg_type = match <u16 as ::util::ser::Readable>::read(&mut reader) {
```

Comment:
Could you make this part of `wire.rs`'s API so as not to leak the wire format across abstraction boundaries?

Reply:
This was pretty half-baked, so removed this commit